appveyor.yml

```diff
@@ -33,6 +33,8 @@ after_build:
   - cmd: 7z a Sunshine-Windows.zip sunshine.exe
   - cmd: 7z a Sunshine-Windows.zip tools\dxgi-info.exe
   - cmd: 7z a Sunshine-Windows.zip tools\audio-info.exe
+  - cmd: 7z a Sunshine-Windows.zip tools\sunshinesvc.exe
+  - cmd: 7z a Sunshine-Windows.zip ..\tools\install-service.bat
```

Comment:
There doesn't appear to be a way to uninstall it

Reply:
Oops, added `uninstall-service.bat`.